documentation/Functions.md

```diff
@@ -402,19 +402,27 @@ adaptTemplatesCobiGen()
 ***
 
 ### createDevon4ngProject <a name="createDevon4ngProject"></a>
+This function is used to create a devon4ng project ib workspace.
```

Comment:
You can additionally link `devon4ng` with the Github repository https://github.com/devonfw/devon4ng

Reply:
done